tokio/src/runtime/metrics/batch.rs

```diff
@@ -32,11 +32,29 @@ pub(crate) struct MetricsBatch {
 
     /// The total busy duration in nanoseconds.
     busy_duration_total: u64,
+
+    /// Instant at which work last resumed (continued after park).
     last_resume_time: Instant,
+
+    /// If `Some`, tracks poll times in `ns`
+    poll_timer: Option<PollTimer>,
+}
+
+struct PollTimer {
+    /// Histogram of mean time spent polling within each band.
```

Comment:
```suggestion
    /// Histogram of time spent polling within each band.
```

There's no mean here, since we're measuring the actual poll time.

Reply:
Sorry, to say it another way: a user _could_ calculate the mean poll times from this data, but the data as it exists is itself not the mean.